src/local-development/types/checksum.types.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Describes the response of API endpoint `v2/sdk/apps/${appName}/${appVersion}/checksum`
+ */
+export interface Checksum {
+	modules: ComponentChecksum[];
+	rpcs: ComponentChecksum[];
+	functions: ComponentChecksum[];
+	accounts: ComponentChecksum[];
+	hooks: ComponentChecksum[];
+	app: ComponentChecksum[];
+}
+
+export interface ComponentChecksum {
+	name: string;
+	checksum: Record<string, string | null>;
```

Comment:
Need documentation, what are keys and what values. Do I understand correctly that it contains checksums of codes and also checksums of metadata somehow? It is not clear. If the stucture has some mix of these information, the @example can make it more clear.

Reply:
Checked, done. :)